service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseManagementOrderService.java

```diff
@@ -77,44 +61,40 @@ public void progressCMOToAction(CaseDetails caseDetails, CaseManagementOrder ord
         return data;
     }
 
-    public CaseManagementOrder buildCMOWithHearingDate(DynamicList list, CaseManagementOrder order) {
-        CaseManagementOrder.CaseManagementOrderBuilder builder = order.toBuilder();
-
-        if (list != null) {
-            HearingDateDynamicElement hearingDateDynamicElement = hearingBookingService.getHearingDynamicElement(list);
+    public OrderAction removeDocumentFromOrderAction(OrderAction orderAction) {
```

Comment:
Technically it doesn't remove any document - it returns new one without the doc. Should the name of method reflect that?

Reply:
`duplicateOrderActionWithoutDocument` maybe?

Reply:
or change the implementation to .toBuilder().doc(null) ?